majordodo-core/src/main/java/majordodo/task/StatusEdit.java

```diff
@@ -316,6 +322,7 @@ public static final StatusEdit WORKER_DIED(String workerId, long timestamp) {
                     doo.writeUTF(taskType);
                     doo.writeInt(maxattempts);
                     doo.writeInt(attempt);
+                    doo.writeLong(requestedStartTime);
```

Comment:
this is going to break compatibility on the format of the log:
- you will not be able to upgrade a broker
- followers will not be able to follow a broker of an previous version

it is important that new followers can follow "old leaders", in a rolling restart you are going to restart followers if possible and than the leader